src/ripple/overlay/README.md

```diff
@@ -111,8 +94,8 @@ Content-Type: application/json
 
 * *HTTP Version*
```

Comment:
I like the idea of a table. I'll see what I can do. Re: xrpl.org, certainly this could also be there, but I think it's good to have these "inline" markdown files (providing we keep them up to date) so that the documentation can travel _with_ the project.